packages/core/strapi/lib/commands/utils/data-transfer.js

```diff
@@ -257,6 +268,79 @@ const getTransferTelemetryPayload = (engine) => {
   };
 };
 
+/**
+ * Get a transfer engine schema diff handler that confirms with the user before bypassing a schema check
+ */
+const getDiffHandler = (engine, { force, action }) => {
+  return async (context, next) => {
+    // if we abort here, we need to actually exit the process because of conflict with inquirer prompt
+    setSignalHandler(async () => {
+      await abortTransfer({ engine, strapi });
+      exitWith(1, exitMessageText(action, true));
+    });
+
+    let workflowsStatus;
+    const source = 'Schema Integrity';
+
+    Object.entries(context.diffs).forEach(([uid, diffs]) => {
+      for (const diff of diffs) {
+        const path = `${uid}.${diff.path.join('.')}`;
+        const endPath = diff.path[diff.path.length - 1];
+
+        // Catch known features
+        if (
+          uid === 'admin::workflow' ||
+          uid === 'admin::workflow-stage' ||
+          endPath.startsWith('strapi_reviewWorkflows_')
```

Comment:
do you think we could document this? or make this extendable from other packages?

It might be useful for others to know this exist too. 
ping @nathan-pichon

Reply:
awesome! thank you :)

Reply:
My initial thought is adding a schema option (available in a content type itself and its attributes) for flagging something as part of a feature, which will group it for reporting like in this case. And then another for if the data should be included in a transfer (and by default excluding anything in 'admin::' that isn't flagged)

But I'll write that all down more clearly in an ADR or RFC once I get the time for it.

Reply:
that sounds like a good idea , will wait for the ADR/RFC :)